test-results/failing/New/table_cells.md

```diff
@@ -72,5 +72,5 @@ Should give output:
 But instead was:
 
 ````````````html
-<p>|1| |-| |1|</p><p>|1| |-| |||</p><p>|1| |-| |1\1|</p><p>|1| |-| |\\||</p><p>|1| |-| |\\||</p><p>|1|2| |-|-| ||2|</p><p>|1|2| |-|-| |1|\|2|\|</p><p>|1|2| |-|-| | |2|</p><p>1|2 -|- 1|\|2|\</p><p>1|2 -|- |2</p><p>1|2 -|- 1|2|</p><p>1|2 -|- 1|2|</p><p>|1|2| |-|-| |1|2||</p><p>|1|2| |-|-| |1|2||</p>
+<table><thead><tr><th>1</th></tr></thead><tbody><tr><td>1</td></tr></tbody></table><table><thead><tr><th>1</th></tr></thead><tbody><tr><td>|</td></tr></tbody></table><table><thead><tr><th>1</th></tr></thead><tbody><tr><td>1\1</td></tr></tbody></table><table><thead><tr><th>1</th></tr></thead><tbody><tr><td>|</td></tr></tbody></table><table><thead><tr><th>1</th></tr></thead><tbody><tr><td>\|</td></tr></tbody></table><table><thead><tr><th>1</th><th>2</th></tr></thead><tbody><tr><td>2</td><td></td></tr></tbody></table><table><thead><tr><th>1</th><th>2</th></tr></thead><tbody><tr><td>1||2||</td><td></td></tr></tbody></table><table><thead><tr><th>1</th><th>2</th></tr></thead><tbody><tr><td></td><td>2</td></tr></tbody></table><table><thead><tr><th>1</th><th>2</th></tr></thead><tbody><tr><td>1||2|\</td><td></td></tr></tbody></table><table><thead><tr><th>1</th><th>2</th></tr></thead><tbody><tr><td></td><td>2</td></tr></tbody></table><table><thead><tr><th>1</th><th>2</th></tr></thead><tbody><tr><td>1</td><td>2|</td></tr></tbody></table><table><thead><tr><th>1</th><th>2</th></tr></thead><tbody><tr><td>1</td><td>2|</td></tr></tbody></table><table><thead><tr><th>1</th><th>2</th></tr></thead><tbody><tr><td>1</td><td>2|</td></tr></tbody></table><table><thead><tr><th>1</th><th>2</th></tr></thead><tbody><tr><td>1</td><td>2|</td></tr></tbody></table>
```

Comment:
Based on babelmark, I think this spec conflicts with GFM on how it handles `\\|`.